Monitoring in a Box/Ansible/ansible.cfg

```diff
@@ -0,0 +1,3 @@
+[defaults]
+host_key_checking = False
+vault_password_file = .vault
```

Comment:
Remember to put the vault password in 1Password and share with the team.

Reply:
okay